tf2_ros/include/tf2_ros/transform_listener.h

```diff
@@ -61,6 +61,7 @@ get_default_transform_listener_sub_options()
     rclcpp::QosPolicyKind::Durability,
     rclcpp::QosPolicyKind::History,
     rclcpp::QosPolicyKind::Reliability};
+  options.use_intra_process_comm = rclcpp::IntraProcessSetting::Disable;
```

Comment:
Can you please put a comment in here explaining why we are doing this?  That way it will be easier for future readers to figure out what happened here.  Especially the fact that we may be able to remove this in the future when intra-process gains more features.  Same goes for the static one below.